dependencies.yaml

```diff
@@ -747,7 +747,7 @@ dependencies:
     common:
       - output_types: [conda, requirements, pyproject]
         packages:
-          - polars>=1.11,<1.15
+          - polars>=1.11,<1.17
```

Comment:
can we support polars 1.17.1 (the latest release)? with <1.18

Reply:
Yes